client/src/components/CaptureMatching/Captures.js

```diff
@@ -0,0 +1,66 @@
+import React, {useState, useEffect} from 'react'
+
+import CaptureImage from './CaptureImage'
+
+import { makeStyles } from '@material-ui/core/styles';
+
+
+
+
+const useStyles = makeStyles({
+    containerBox: {
+        margin: '10px',
+        padding: '10px',
+    },
+
+    headerIconBox: {
+        marginLeft: '33px',
+        marginTop: '20px'
+
+    }
+})
+
+function Captures(props) {
+
+    const classes = useStyles()
+    const treeList = props.treeList
+
+    const {
+        imageData,
+        loading,
+        currentPage,
+        noOfPages,
+        handleChange,
+        captureApiFetch,
+        imgPerPage,
+        skipCapture,
+        imgCount,
+        handleSkip
+    } = props;
+
+
+    return (
+
+
+        <div style={{width: '50%'}}>
```

Comment:
The `width` property should be applied in `CaptureMachineFrame`, since that's where you know how much space is available, and it's clearer that it's a split screen when next to `width="50%"` for the candidate side of the tool.